docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/upgrade-guides/6-0-0-upgrade-guide/index.md

```diff
@@ -0,0 +1,153 @@
+
+```json
+{
+   // 1-0-0
+   "properties": {
+      "a": {"type": "integer"}
+   }
+}
+```
+
+```json
+{
+   // 1-0-1
+   "properties": {
+      "b": {"type": "integer"}
+   }
+}
+```
+
+These would be merged into the following:
+```json
+{
+   // merged
+   "properties": {
+      "a": {"type": "integer"},
+      "b": {"type": "integer"}
+   }
+}
+```
+
+
+Second, the loader does not fail when it can’t modify the database table to store both old and new events. (As a reminder, an example would be changing the type of a field from `integer` to `string`.) Instead, it creates a table for the new data as an exception. The users can then run SQL statements to resolve this situation as they see fit. For instance, consider these two schemas:
+```json
+{
+   // 1-0-0
+   "properties": {
+      "a": {"type": "integer"}
+   }
+}
+```
+
+```json
+{
+   // 1-0-1
+   "properties": {
+      "a": {"type": "string"}
+   }
+}
+```
+
+Because `1-0-1` events cannot be loaded into the same table with `1-0-0`, the data would be put in a separate table, e.g. `com_snowplowanalytics_ad_click_1_0_1_recovered_9999999`, where:
+  - `1_0_1` is the version of the offending schema;
+  - `9999999` is a hash code unique to the schema (i.e. it will change if the schema is overwritten with a different one).
+
+If you create a new schema `1-0-2` that reverts the offending changes and is again compatible with `1-0-0`, the data for events with that schema will be written to the original table as expected.
+
+### Identification of the schemas that need patching
+
+After the rdb-loader's upgrade, users might find out that some old schema lineages land in the recovery table.
+
+To avoid this, older iglu schemas must be patched to align with the latest.
+
+To help with the identification of the schemas that need patching, we have rewritten the rdbms table-check command.
+`igluctl` will now cross-check the types between the iglu server and RedShift or Postgres. Depending on what types
+offended the schema errors might vary.
+
+Procedure of identifying offending warehouse patches is as follows:
+
+1) Run `igluctl static generate` command. If recovery table to be created it will show up as a warning message. Example:
+```bash
+mkdir <schemas_folder> <sql_folder>
+igluctl static pull <schemas_folder> <iglu_url> <iglu_key>
+igluctl static generate <schemas_folder> <sql_folder> 
+# ...
+# iglu:com.snowplowanalytics.iglu/resolver-config/jsonschema/1-0-3 has a breaking change Incompatible encoding in column cache_size old type RedshiftBigInt/ZstdEncoding new type RedshiftDouble/RawEncoding
+# ...
+```
+
+2) Run `table-check` to crosscheck if warehouse was patched. This is done in case if  schema lineage had a broken schema
+   that were supreseed and no longer used, for example:
```

Comment:
```suggestion
2) Run the `igluctl table-check` command to check if the warehouse was patched. This is done in case there are incorrectly evolved schema versions that were never used, for example:
```